doc/nrf/known_issues.rst

```diff
@@ -667,14 +848,6 @@ Subsystems
 Bluetooth® LE
 =============
 
-.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
```

Comment:
(commenting this line as the new entry should go to the Bluetooth section probably), please add something like:
"NCSDK-12886: Peripheral UART sample built for nRF52811 asserts on the nRF52840 DK in rev. 2.1.0 (build target: nrf52840dk_nrf52811)"